README.md

```diff
@@ -28,118 +28,96 @@
-
-**Why setting a single priority?**
-
-The 2019 priority is geared to grow our maturity as a project, increase our adoption and community, and set ourselves up for the future. To be successful, we need to level-up our ability to land complete, polished, and accessible improvements that meet our user's needs for stability, scalability, privacy, and security. We need to keep growing intelligently, both by the types of communities that can help us accelerate growth and by the number of users they can help us reach. And we need to set ourselves up with the understanding and capabilities to reach our future goals for the project and scale our impact. We are confident that by aligning on *one* ambitious priority, we will be leveling up the core baseline of the IPFS Protocol, making it ready to tackle all the other goals that follow. 
-
-To help us identify our top priority, we designed a sorting function to prioritize goals that 1) we can land end-to-end in 2019, 2) will bring new partners and users into the community, and 3) will set us up for future goals while delivering near-term value. After sorting, we identified what was the most achiavable (lower D) with largest Ecosystem Growth (higher E) and that would translate into Impact to our Ecosystems and Project (higher I).
+### Scoping in to 2020 H1
+Instead of a 2020 plan, we decided to focus on a **2020 H1 plan (covering Q1 & Q2)** so as to: 
+- Enable our team to truly focus on one thing, complete it, and then move on to other challenges instead of doing many things at once
+- Better understand the components of each goal and plan our time accordingly to hit them by not trying to nail down plans too far into the future
+- Reduce the chance of surprises, re-prioritizations, or market shifts that require us to refocus energy or change our plan
 
-**Sorting Function Explanation**
+### 2020 H1 Priority Selection Criteria
+Before selecting a 2020 H1 priority, we did an open call for [Theme Proposals](https://github.com/ipfs/roadmap/blob/master/2020-IPFS-Project-Planning.md) to surface areas the community felt were of high importance and urgency. We combined these great proposals with an analysis of the project, team, and ecosystem state - and the biggest risks to IPFS Project success. Out of that analysis, we identified there were two main aspects our 2020 H1 plan **MUST** address:
+1. Mitigate current IPFS pain points around network performance and end user experience that are hindering wider adoption and scale
+1. Increase velocity, alignment, and capacity for IPFS devs and contributors to ensure our time and efforts are highly leveraged
```

Comment:
The second point feels vague for a "must".

Here's a take on breaking it down a bit:

* We want the core team and broader community to understand the content routing need, so that we can all be aligned around it
* We want *how* we work as an open source project to be clear, accessible, responsive, fun
* We think if achieve the two prior points, we'll hit the content routing goal sooner
* We can then broaden adoption efforts like new features, more integrations, broader outreach, etc

Does that sound close as an unpacked version of Must #2?

Reply:
Added a descriptor on this. I think that breakdown is pretty accurate for what we should do to achieve this MUST with the content routing goal, but decision making "must" was actually less specific than that to account for many possible solutions -- more around finding ways to energize and increase velocity in our work and that of the entire community